cyclops/tasks/cxr_classification.py

```diff
@@ -0,0 +1,323 @@
+"""Chest X-ray Classification Task."""
+import logging
+from functools import partial
+from typing import Any, Dict, List, Optional, Sequence, Tuple, Union, get_args
+
+import numpy as np
+from datasets import Dataset
+from monai.transforms import Compose
+
+from cyclops.datasets.slicing import SlicingConfig as SliceSpec
+from cyclops.evaluate.evaluator import Evaluator
+from cyclops.evaluate.metrics.metric import Metric, MetricCollection, create_metric
+from cyclops.models.catalog import _img_model_keys
+from cyclops.models.utils import get_device
+from cyclops.models.wrappers import WrappedModel
+from cyclops.tasks.util import apply_image_transforms
+from cyclops.utils.log import setup_logging
+
+# from cyclops.datasets.slice import SliceSpec XXX: Add when the branch is merged
+
+
+LOGGER = logging.getLogger(__name__)
+setup_logging(print_level="INFO", logger=LOGGER)
+
+
+class CXRClassification:
+    """Chest X-ray classification task modeled as a multi-label classification task."""
+
+    def __init__(
+        self,
+        models: Union[WrappedModel, Sequence[WrappedModel], Dict[str, WrappedModel]],
+        task_features: List[str],
```

Comment:
How about specifying default values? Like this: https://github.com/huggingface/datasets/blob/main/src/datasets/tasks/image_classification.py#L10